DataFormats/Detectors/TRD/src/RawData.cxx

```diff
@@ -322,6 +324,10 @@ void printHalfCRUHeader(o2::trd::HalfCRUHeader& halfcru)
   LOG(info) << "Raw: " << std::hex << halfcru.word0 << " " << halfcru.word12[0] << " " << halfcru.word12[1] << " " << halfcru.word3 << " " << halfcru.word47[0] << " " << halfcru.word47[1] << " " << halfcru.word47[2] << " " << halfcru.word47[3];
 }
 
+void printDigitADCMask(o2::trd::DigitMCMADCMask& mask)
+{
+}
```

Comment:
Remove this empty method?

Reply:
oops

Reply:
removed we already have printDigitMCMADCMask, not sure what that was left in.